### ISSUE TITLE
Field normalization

### ISSUE THREAD

**Issue by dantleech** [bolt/bolt#6628]
For *some reason*, some image fields are being persisted without the `file` attribute, which causes the following exception when accessing the containing content in the backoffice:

```
Key "file" for array with keys "title, alt" does not exist in "@bolt/editcontent/fields/_image.twig" at line 49.
```

Is it possible to add some normalization in the `Field\Type\ImageType` when persisting + hydrating the field? e.g. :

```php
$data = array_merge(array(
    'title' => null,
    'alt' => null,
    'file' => null,
), $data);
```

the following patch is a more hacky way to stop the edit page crashing:

```diff
diff --git a/app/view/twig/editcontent/fields/_image.twig b/app/view/twig/editcontent/fields/_image.twig
index a8aee241f..9a7645df0 100644
--- a/app/view/twig/editcontent/fields/_image.twig
+++ b/app/view/twig/editcontent/fields/_image.twig
@@ -46,7 +46,7 @@
     preview: {
         'alt':          'Preview',
         'height':       preview_h,
-        'src':          image.file is not empty ? image.file|thumbnail(preview_w, preview_h, 'b') : app.paths.app ~ 'view/img/default_empty_4x3.png',
+        'src':          (image.file is defined and image.file is not empty) ? image.file|thumbnail(preview_w, preview_h, 'b') : app.paths.app ~ 'view/img/default_empty_4
         'width':        preview_w,
     },
 }
```

**Comment by GawainLynch** [comment]
Fixed in #6512

**Comment by dantleech** [comment]
You're a :star: , thanks.
